cob_description/urdf/cob4_base/base.urdf.xacro

```diff
@@ -19,6 +19,9 @@
 
   <xacro:macro name="base" params="name use_old_joint_names">
 
+    <xacro:property name="drive_vel" value="19.95" />
+    <xacro:property name="steer_vel" value="117.81" />
```

Comment:
SI units for rotation -> rad/s ;)
I got the values from @ipa-tif

If we have the number for the raws, we can add them as well.
And remove the "alpha tested default" of 100.